# Optimizable weights contains the list of weights we want to optimize, their bounds and their
# reconfigure name. This file is unused and serves as a template with all posible tuning parameters.

weights:
  Q1:
    lower_bound: 0.0
    upper_bounds: 1000
    initial_value: 100.0
  Q2:
    lower_bound: 0.0
    upper_bounds: 1000
    initial_value: 100.0
  R1:
    lower_bound: 0.0
    upper_bounds: 1000
    initial_value: 0.2
  R2:
    lower_bound: 0.0
    upper_bounds: 1000
    initial_value: 0.3
  q:
    lower_bound: 0.0
    upper_bounds: 1000
    initial_value: 3.5
  lag_compensation_time:
    lower_bound: 0.0
    upper_bounds: 1
    initial_value: 0.03
  Df:
    lower_bound: 0.2
    upper_bounds: 3.0
    initial_value: 0.65
  Dr:
    lower_bound: 0.2
    upper_bounds: 3.0
    initial_value: 1.0
  Cf:
    lower_bound: 1.0
    upper_bounds: 2.0
    initial_value: 1.5
  Cr:
    lower_bound: 1.0
    upper_bounds: 2.0
    initial_value: 1.45
  Bf:
    lower_bound: 3.0
    upper_bounds: 10.0
    initial_value: 5.2
  Br:
    lower_bound: 3.0
    upper_bounds: 10.0
    initial_value: 8.5

reconfigure_names:
    - '/car_1/controller_node'
